src/Detached.Mappers/Annotations/AbstractAnnotationHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Detached.Mappers.Types.Class.Builder;
+
+namespace Detached.Mappers.Annotations
```

Comment:
introduced a way to mark a non abstract class as abstract for the mapper
(maybe I should rename all terms as DynamicType instead of Abstract)